src/validate-voter.php

```diff
@@ -16,8 +16,12 @@
 	$voter_query = "SELECT * FROM voter WHERE voter_id = $voter_id";
 	$result = $conn->query($voter_query);
 	$row = $result->fetch_assoc();
```

Comment:
Use prepared statement next time even though this is not a form submission — just for consistency and a bit of false sense of security. LOL